client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -16,6 +16,9 @@ import { isFrontPage, isPostsPage } from 'state/pages/selectors';
 import PostRelativeTimeStatus from 'my-sites/post-relative-time-status';
 import canCurrentUser from 'state/selectors/can-current-user';
 import getEditorUrl from 'state/selectors/get-editor-url';
+import PostMetadata from 'lib/post-metadata';
```

Comment:
@Automattic/team-calypso did we want to keep this pattern? Feels like we can take advantage of an apiTransform instead in the data pipeline, or just let a selector handle it.